google-sheets/info.json

```diff
@@ -0,0 +1,1148 @@
+        "tooltip": "Unique Client Secret of the Google Sheets that is used to create an authentication token required to access the API."
+      },
+      {
+        "title": "Authorization Code",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "code",
+        "tooltip": "The authorization code that you acquired during the authorization step. For more information, see the Accessing the Google Sheets API section.",
+        "description": "The authorization code that you acquired during the authorization step. For more information, see the Accessing the Google Sheets API section."
+      },
+      {
+        "title": "Redirect URL",
+        "description": "The redirect_uri of your app, where authentication responses can be sent and received by your app. It must exactly match one of the redirect_uri's you registered in the app registration portal.",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "redirect_url",
+        "placeholder": "E.g. https://localhost/myapp",
+        "tooltip": "The redirect_uri of your app, where authentication responses can be sent and received by your app. It must exactly match one of the redirect_uri's you registered in the app registration portal."
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True.",
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "create_spreadsheet",
+      "title": "Create Spreadsheet",
+      "description": "Creates a spreadsheet, returning the newly created spreadsheet based on the input parameters you have specified.",
+      "category": "investigation",
+      "annotation": "create_spreadsheet",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Title",
+          "name": "title",
+          "description": "Specify the title of the spreadsheet based on which you want to create.",
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Specify the title of the spreadsheet based on which you want to create."
+        },
+        {
+          "title": "Locale/Language",
+          "name": "locale",
+          "description": "Specify the locale of the spreadsheet in one of the following formats: en, fil, or en_US. Note: When updating this field, not all locales/languages are supported.",
```

Comment:
if possible you can specify the link where user can find all locales/languages

Reply:
We don't have any table or link.